pkgs/development/interpreters/ruby/default.nix

```diff
@@ -49,12 +49,12 @@ let
           rev    = tag;
           sha256 = sha256.git;
         } else fetchurl {
-          url = "http://cache.ruby-lang.org/pub/ruby/${ver.majMin}/ruby-${ver}.tar.gz";
```

Comment:
Actually this works for me, if you take a look at `ruby-version.nix`, you see  `__toString`.
If I add:

```
url = lib.traceVal "http://cache.ruby-lang.org/pub/ruby/${ver.majMin}/ruby-${ver}.tar.gz";
```

I see:

```
trace: http://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.4.tar.gz
```

Could this come from using an old version of nix?

cc: @zimbatm